packages/kit/test/apps/basics/test/test.js

```diff
@@ -620,6 +619,21 @@ test.describe.parallel('Errors', () => {
 			expect(await page.innerHTML('h1')).toBe('401');
 		}
 	});
+
+	test('custom error page is rendered if handle hook throws an error', async ({
+		baseURL,
+		page,
+		javaScriptEnabled,
+		started
```

Comment:
if you rebase against `master` there is no `started` anymore. You should rebase and then delete this, the `await started()`, and the `waitForTimeout` lines

Reply:
Thank you for the hint. I thought that I already had rebased, but that must have been another branch I was working on ... 🤦

Reply:
Now it works. The failing test on windows is also failing in other PRs ... so this is probably not the fault of my changes, right?